prow/plugins/welcome/welcome_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
-// TODO(bentheelder): these tests are a bit lame.
+// TODO: these tests are a bit lame.
```

Comment:
I feel like you should go ahead and drop this TODO, it's too subjective

Reply:
done